swarm/fuse/swarmfs_test.go

```diff
@@ -866,21 +960,38 @@ func TestFUSE(t *testing.T) {
 	}
 	ta := &testAPI{api: api.NewApi(dpa, nil, nil)}
 
-	t.Run("mountListAndUmount", ta.mountListAndUnmount)
-	t.Run("maxMounts", ta.maxMounts)
-	t.Run("remount", ta.remount)
-	t.Run("unmount", ta.unmount)
-	t.Run("unmountWhenResourceBusy", ta.unmountWhenResourceBusy)
-	t.Run("seekInMultiChunkFile", ta.seekInMultiChunkFile)
-	t.Run("createNewFile", ta.createNewFile)
-	t.Run("createNewFileInsideDirectory", ta.createNewFileInsideDirectory)
-	t.Run("createNewFileInsideNewDirectory", ta.createNewFileInsideNewDirectory)
-	t.Run("removeExistingFile", ta.removeExistingFile)
-	t.Run("removeExistingFileInsideDir", ta.removeExistingFileInsideDir)
-	t.Run("removeNewlyAddedFile", ta.removeNewlyAddedFile)
-	t.Run("addNewFileAndModifyContents", ta.addNewFileAndModifyContents)
-	t.Run("removeEmptyDir", ta.removeEmptyDir)
-	t.Run("removeDirWhichHasFiles", ta.removeDirWhichHasFiles)
-	t.Run("removeDirWhichHasSubDirs", ta.removeDirWhichHasSubDirs)
-	t.Run("appendFileContentsToEnd", ta.appendFileContentsToEnd)
+	t.Run("mountListAndUmountEncrypted", ta.mountListAndUnmountEncrypted)
+	t.Run("mountListAndUmountNonEncrypted", ta.mountListAndUnmountNonEncrypted)
```

Comment:
A typo in two Run name `Umount` in two runs. I suppose that it should be `Unmount` as it is the function name that is testing, as well.

Reply:
Thanks, fixed in e0294b5560f930d5e60af5ade45befab35acdb5f